src/main/java/gov/nasa/pds/api/engineering/elasticsearch/ElasticSearchUtil.java

```diff
@@ -24,6 +26,14 @@ static public String jsonPropertyToElasticProperty(String jsonProperty) {
 		
 	}
 	
+	static private void append (ArrayList<Reference> to, String ID, String baseURL)
```

Comment:
Rename the method more explocitly like addReference,  or appendToReference

Reply:
Done